data/synapse-data-mongodb/README.md

```diff
@@ -0,0 +1,31 @@
+# synapse-data-mongodb
```

Comment:
Maybe put in the readme the required properties; e.g, spring.data.mongodb.database

Reply:
done